source/_data/glossary.yml

```
@@ -311,26 +311,20 @@
-    Assistant Core installation method) or as a container using Docker
-    (typically referred to as Home Assistant Container installation method).
-  link: /installation/#advanced-installation-methods
-  excerpt: >
-    Home Assistant Core is the heart of Home Assistant itself. It is a Python
-    program that powers every installation type, but can be installed
-    standalone.
+    is part of all installation types. It can be installed standalone (without Home Assistant
+    Supervisor) as a container using Docker (this is typically referred to as the Home Assistant
+    Container installation method). For development, Core can also be run using a Virtual
+    Environment (previously referred as the Home Assistant Core installation method.
+    For production setup, the [Home Assistant Core installation method is deprecated](https://www.home-assistant.io/blog/2025/05/22/deprecating-core-and-supervised-installation-methods-and-32-bit-systems/). 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)